src/monitor-mint-calls/monitor-mint-calls.spec.js

```diff
@@ -3,16 +3,16 @@ const { getAbi, getAddress } = require('@uma/contracts-node');
 const { createTransactionEvent } = require('forta-agent');
 
 // override the timeout from the agent configuration JSON file before the handler is imported
-jest.mock('../agent-config.json', () => ({
+jest.mock('../../agent-config.json', () => ({
   monitorMintCalls: { rpcTimeoutMilliseconds: 1000 },
 }));
```

Comment:
These lines should be deleted.  The variable `rpcTimeoutMilliseconds` is a vestige of when the `monitor-mint-calls` handler was directly calling out to the Alchemy JSON-RPC API to get trace data instead of specifying the `traceRpcUrl` value in the `forta.config.json` file.
```suggestion
```